sqlstore/src/main/resources/db/migration/V8844__paritions_for_platform.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE Platform_Sizes (
```

Comment:
Table names with underscores are usually join tables in MISO. Should be "PlatformSizes" to be more consistent